src/Orikivo/Services/TemplateBuilder.cs

```diff
@@ -326,7 +331,7 @@ public static Bitmap BuildNewLevelTemplate(Bitmap userAvatar)
                 }
 
 
-                if (char.IsWhiteSpace(secondaryGameName.FirstOrDefault()))
+                if (!string.IsNullOrWhiteSpace(secondaryGameName))
```

Comment:
Change to `secondaryGameName.StartsWith(' ')`, as this doesn't really check the first character, but the entire string